RESTserver/src/main/java/com/softserve/if072/restservice/service/StoreService.java

```diff
@@ -31,8 +32,14 @@ public StoreService(StoreDAO storeDAO) {
     @Transactional
     public List<Store> getAllStores(int userId) throws DataNotFoundException {
         List<Store> stores = storeDAO.getAllStoresByUser(userId);
+        List<Store> enabledStores = new ArrayList<>();
         if (!stores.isEmpty()) {
-            return stores;
+            for (Store getStore : stores) {
+                if (getStore.isEnabled()) {
+                    enabledStores.add(getStore);
+                }
+            }
+            return enabledStores;
```

Comment:
Don't do in this way.

Add new method into DAO, which return all enabled stores